examples/apps/heart_disease_eval/workload/heart_disease_evaluation_logic.cpp

```diff
@@ -242,31 +235,38 @@ std::string HeartDiseaseEvalLogic::executeWorkOrder(
             dataString = inputString[1];
 
         std::vector<std::string> medData = split(dataString, ' ');
-        if (medData.size() != 14)
-            return "Error with missing or incorrect input format";
-
-        int risk = 100 -
-            int( score_age(std::stoi(medData[0])) * 0.03
-            + score_sex(std::stoi(medData[1])) * 0.01
-            + score_cp(std::stoi(medData[2])) * 0.21
-            + score_trestbps(std::stoi(medData[3])) * 0.05
-            + score_chol(std::stoi(medData[4])) * 0.05
-            + score_fbs(std::stoi(medData[5])) * 0.04
-            + score_restecg(std::stoi(medData[6])) * 0.19
-            + score_thalach(std::stoi(medData[7])) * 0.06
-            + score_exang(std::stoi(medData[8])) * 0.18
-            + score_oldpeak(std::stoi(medData[9])) * 0.05
-            + score_slop(std::stoi(medData[10])) * 0.03
-            + score_ca(std::stoi(medData[11])) * 0.04
-            + score_thaldur(std::stoi(medData[12])) * 0.02
-            + score_num(std::stoi(medData[13])) * 0.04 );
-
-        // Update accumulation
-        totalRisk += risk;
-        count++;
-        // Use accumulated data to calculate the result
-        resultString = "You have a " +
-            std::to_string(totalRisk/count) + "% risk of heart disease";
+        if (medData.size() != 14) { // return accumulated calculations
```

Comment:
Earlier this check was used to validate the input format. With this change if input format is incorrect ( not passing required number of input values), we are not returning error message. What is the expected way to use Heart disease workload with command line (eg: with generic client).

Reply:
When the worker receives 0 data items it returns an average risk.
When it receives 14 data items it returns the individual risk.
I will change the code to check for either 0 or 14 data items and return an error for other counts.